### PR TITLE
Update convertCSVtoJSON.js

### PR DESCRIPTION
**Pull request by arvi18**
- kuch bhi

### CODE REVIEW

**Comment by arvi18** [comment]
normie comment

File: convertCSVtoJSON.js [hunk 1]
```diff
@@ -32,3 +32,4 @@ axios
   .catch((error) => {
```
Comment (arvi18) [inline convertCSVtoJSON.js:32]: commits section single comment

File: convertCSVtoJSON.js [hunk 2]
```diff
@@ -32,3 +32,4 @@ axios
...
     console.error("Error fetching CSV data:", error);
   });
```
Comment (arvi18) [inline convertCSVtoJSON.js:34]: another commit sec

File: convertCSVtoJSON.js [hunk 3]
```diff
@@ -32,3 +32,4 @@ axios
   .catch((error) => {
     console.error("Error fetching CSV data:", error);
```
Comment (arvi18) [inline convertCSVtoJSON.js:33]: files changes

**Review by arvi18 — COMMENTED** [review]
finsih with comment

**Review by gandalf-repo — COMMENTED** [review]
comment hehe

**Review by gandalf-repo — CHANGES_REQUESTED** [review]
change karo

**Review by gandalf-repo — APPROVED** [review]
approve kela

**Review by arvi18 — COMMENTED** [review]
author comments

**Review by gandalf-repo — COMMENTED** [review]
gandlaf says somehting

**Review by gandalf-repo — APPROVED** [review]
approve kela

**Pull request closed without merging**